### PR TITLE
Prevent possible notice if image doesn't exist

### PR DESCRIPTION
**Pull request by JoryHogeveen**
## Description

Fixes: #5866 

## PR Checklist

- [x] My code is tested.
- [x] My code follows the WordPress code style.
- [x] My code follows has proper inline documentation.

### CODE REVIEW

File: includes/media.php [hunk 1]
```diff
@@ -88,12 +88,14 @@ function pods_image( $image, $size = 'thumbnail', $default = 0, $attributes = ''
 	$default = pods_image_id_from_field( $default );
 
 	if ( 0 < $id ) {
-		if ( $force ) {
+		if ( $force && 'full' !== $size ) {
 			$full = wp_get_attachment_image_src( $id, 'full' );
-			$src  = wp_get_attachment_image_src( $id, $size );
+			if ( ! empty( $full[0] ) ) {
+				$src  = wp_get_attachment_image_src( $id, $size );
```
Comment (tr1b0t) [inline includes/media.php:94]: :warning: Equals sign not aligned correctly; expected 1 space but found 2 spaces

<sub>hash: e33a73278c2abcca849d57ce5bbef116</sub>

File: includes/media.php [hunk 2]
```diff
@@ -140,12 +142,14 @@ function pods_image_url( $image, $size = 'thumbnail', $default = 0, $force = fal
...
```
Comment (tr1b0t) [inline includes/media.php:148]: :warning: Equals sign not aligned correctly; expected 1 space but found 2 spaces

<sub>hash: 96b538570746d1f44470979393e3d3f5</sub>

**Review by tr1b0t — COMMENTED** [review]
:warning: 2 warning(s)

File: includes/media.php [hunk 3]
```diff
@@ -246,6 +236,32 @@ function pods_attachment_import( $url, $post_parent = null, $featured = false )
 	return $attachment_id;
 }
 
+/**
+ * Resize an image on if it doesn't exist.
+ *
+ * @param int          $attachment_id Attachment ID
```
Comment (tr1b0t) [inline includes/media.php:242]: :no_entry_sign: Parameter comment must end with a full stop

<sub>hash: 2d0dacfea85261f5802186f6408677ec</sub>

File: includes/media.php [hunk 4]
```diff
@@ -246,6 +236,32 @@ function pods_attachment_import( $url, $post_parent = null, $featured = false )
...
+ * @param string|array $size          Size to be generated
```
Comment (tr1b0t) [inline includes/media.php:243]: :no_entry_sign: Parameter comment must end with a full stop

<sub>hash: ff697304f789ceccf5a4961116ff6aa9</sub>

File: includes/media.php [hunk 5]
```diff
@@ -246,6 +236,32 @@ function pods_attachment_import( $url, $post_parent = null, $featured = false )
...
+ * @return boolean Image generation result
+ *
+ * @since 2.7.23
+ */
+function pods_maybe_image_resize( $attachment_id, $size ) {
+
+	if ( 'full' !== $size ) {
+		$full = wp_get_attachment_image_src( $attachment_id, 'full' );
+		if ( ! empty( $full[0] ) ) {
+			$src  = wp_get_attachment_image_src( $attachment_id, $size );
```
Comment (tr1b0t) [inline includes/media.php:254]: :warning: Equals sign not aligned correctly; expected 1 space but found 2 spaces

<sub>hash: 58dff66a0aba986b6c77af18cdb44dbd</sub>

**Review by tr1b0t — COMMENTED** [review]
:no_entry_sign: 2 error(s)
:warning: 1 warning(s)

File: includes/media.php [hunk 6]
```diff
@@ -247,12 +237,38 @@ function pods_attachment_import( $url, $post_parent = null, $featured = false )
...
+ *
+ * @param int          $attachment_id Attachment ID.
+ * @param string|array $size          Size to be generated.
+ *
+ * @return boolean Image generation result.
+ *
+ * @since 2.7.23
+ */
+function pods_maybe_image_resize( $attachment_id, $size ) {
+
```
Comment (sc0ttkclark) [inline includes/media.php:250]:
```suggestion
```

File: includes/media.php [hunk 7]
```diff
@@ -247,12 +237,38 @@ function pods_attachment_import( $url, $post_parent = null, $featured = false )
...
+		$full = wp_get_attachment_image_src( $attachment_id, 'full' );
+		if ( ! empty( $full[0] ) ) {
+			$src = wp_get_attachment_image_src( $attachment_id, $size );
+
+			if ( empty( $src[0] ) || $full[0] == $src[0] ) {
+				return pods_image_resize( $attachment_id, $size );
+			}
+		}
+	}
+	// No resize needed.
```
Comment (sc0ttkclark) [inline includes/media.php:261]:
```suggestion

	// No resize needed.
```

File: includes/media.php [hunk 8]
```diff
@@ -247,12 +237,38 @@ function pods_attachment_import( $url, $post_parent = null, $featured = false )
...
+ *
+ * @return boolean Image generation result.
+ *
+ * @since 2.7.23
+ */
+function pods_maybe_image_resize( $attachment_id, $size ) {
+
+	if ( 'full' !== $size ) {
+		$full = wp_get_attachment_image_src( $attachment_id, 'full' );
+		if ( ! empty( $full[0] ) ) {
```
Comment (sc0ttkclark) [inline includes/media.php:253]:
```suggestion

		if ( ! empty( $full[0] ) ) {
```

**Review by sc0ttkclark — APPROVED** [review]

**Pull request merged**